CMakeLists.txt

```diff
@@ -7,11 +7,19 @@ set(PACKAGE_VERSION "1.0.0")
 list(INSERT CMAKE_MODULE_PATH 0 "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules")
 find_package(Poco COMPONENTS Foundation)
 
-if(NOT Poco_FOUND)
+
+function(get_poco)
+  set(options)
+  set(oneValueArgs BUILD_TYPE)
+  set(multiValueArgs)
+  cmake_parse_arguments(get_poco "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN} )
```

Comment:
nitpick: extra space before `)`

I'd remove it myself, but I didn't know if it was intentional or something.

Reply:
Unintentional. Removed in 6ae93dcd7b5be88f2ec029e94ae19aaec35adcaf